pom.xml

```diff
@@ -200,6 +199,12 @@ under the License.
         <version>${hadoop.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.apache.hadoop</groupId>
+        <artifactId>hadoop-hdfs</artifactId>
```

Comment:
Looks like this is a duplicate dependency entry (`hadoop-hdfs` is also declared further down in this file, on line 527)